### PR TITLE
:book: Update documentation according to clusterctl sub command changes

### LINKED ISSUE #1086: update clusterctl cluster resource generation instruction
There are some changes on the `clusterctl` command options:
Example

`clusterctl config cluster ....." ` is replaced by 
`clusterctl generate cluster ...`

Some instructions on the capo need an update accordingly.

### PR DESCRIPTION
**Pull request by Xenwar**
This PR updates documentation of `clusterctl` commands inline to changes on the sub commands to it.

fixes #1086

### CODE REVIEW

**Comment by jichenjc** [comment]
/lgtm

Thanks for the update :)

File: test/e2e/shared/cluster.go [hunk 1]
```diff
@@ -53,7 +53,7 @@ func createClusterctlLocalRepository(config *clusterctl.E2EConfig, repositoryFol
 	createRepositoryInput.RegisterClusterResourceSetConfigMapTransformation(ccmPath, CCMResources)
 
 	clusterctlConfig := clusterctl.CreateRepository(context.TODO(), createRepositoryInput)
-	Expect(clusterctlConfig).To(BeAnExistingFile(), "The clusterctl config file does not exists in the local repository %s", repositoryFolder)
+	Expect(clusterctlConfig).To(BeAnExistingFile(), "The clusterctlConfig file does not exists in the local repository %s", repositoryFolder)
```
Comment (tobiasgiese) [inline test/e2e/shared/cluster.go:56]: WDYT about

```suggestion
	Expect(clusterctlConfig).To(BeAnExistingFile(), "The clusterctl generate file %q does not exists in the local repository %s", clusterctlConfig, repositoryFolder)
```

or

```suggestion
	Expect(clusterctlConfig).To(BeAnExistingFile(), "The clusterctl generate file does not exists in the local repository %s", repositoryFolder)
```

Tbh, I don't like variable names in log output :)

**Review by tobiasgiese — COMMENTED** [review]
one tiny nit 👌🏻 Thanks!
/lgtm

File: test/e2e/shared/cluster.go (hunk 1 above)
Comment (tobiasgiese) [reply test/e2e/shared/cluster.go:56]: Another solution could be

```suggestion
	Expect(clusterctlConfig).To(BeAnExistingFile(), "The clusterctl generate file %q does not exists", clusterctlConfig)
```

Comment (Xenwar) [reply test/e2e/shared/cluster.go:56]: @obiasgiese
The `The clusterctl config file does not ....` is a typo. It was referring to the file mentioned in the `Expect(....)`. 
I was just fixing that. Not related to the actual `clusterctl config `command.

Comment (tobiasgiese) [reply test/e2e/shared/cluster.go:56]: Ah, sure. Didn't pay attention to the context. Thanks for clarification ;) 👍🏻

**Pull request merged**